statistics-dashboard-service/src/main/java/eu/europeana/statistics/dashboard/service/server/StatisticsServer.java

```diff
@@ -0,0 +1,195 @@
+package eu.europeana.statistics.dashboard.service.server;
+
+import eu.europeana.statistics.dashboard.common.api.request.StatisticsFilteringRequest;
+import eu.europeana.statistics.dashboard.common.api.request.StatisticsRangeFilter;
+import eu.europeana.statistics.dashboard.common.api.response.BreakdownResult;
+import eu.europeana.statistics.dashboard.common.api.response.FilteringOptions;
+import eu.europeana.statistics.dashboard.common.api.response.FilteringResult;
+import eu.europeana.statistics.dashboard.common.api.response.ResultListFilters;
+import eu.europeana.statistics.dashboard.common.api.response.StatisticsResult;
+import eu.europeana.statistics.dashboard.common.iternal.FacetValue;
+import eu.europeana.statistics.dashboard.service.exception.FacetDeclarationFailException;
+import eu.europeana.statistics.dashboard.service.utils.RequestUtils;
+import eu.europeana.statistics.dashboard.common.iternal.FieldMongoStatistics;
+import eu.europeana.statistics.dashboard.service.persistence.MongoSDDao;
+import eu.europeana.statistics.dashboard.service.persistence.StatisticsData;
+import eu.europeana.statistics.dashboard.service.persistence.StatisticsQuery;
+import eu.europeana.statistics.dashboard.service.persistence.StatisticsQuery.ValueRange;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.springframework.stereotype.Component;
+
+/**
+ * This class is responsible for retrieving the statistics data that is requested
+ */
+@Component
+public class StatisticsServer {
```

Comment:
During design, it was suggested to have this name since it *serves* the data. 
However, what you're saying is also a good point, to keep the structure consistent throughout projects.
I can make this change for consistency purposes, but what do you think?

Reply:
I would still stick to the consistent naming.
For an argument I could say that it's a _service_ that _serves_ the data, practically identifying as the same thing.
And it would probably be under the package `eu.europeana.statistics.dashboard.service` with the package `eu.europeana.statistics.dashboard.service.server` removed.

Reply:
Ok then, I'll apply this change